src/Table/tableUtil.tsx

```diff
@@ -0,0 +1,19 @@
+import { StyledArrowDown, StyledArrowUp, StyledUnsorted } from './tableStyles';
+
+export interface Column {
+    disableSortBy: boolean;
+    isSorted: boolean;
+    isSortedDesc: boolean;
+}
+
+export const getSortBySVG = ({disableSortBy, isSorted, isSortedDesc}: Column) => {
+    if(disableSortBy){
```

Comment:
nit: add proper spacing on this line (e.g. `if (disableSortBy) {`.
Similar comment above in destructured variables.